GR.Laromedel.AccessPortal.Web/src/footer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function Footer() {
+  return (
+    <div className="h-40 pt-6 bg-green-700">
+      <div className="pl-6 w-256 mx-auto">
```

Comment:
Maybe I'm being a bit nitpicky now, but I think it would be better to use the existing `Container` component here to avoid specifying the page width in multiple places.

Reply:
No that's a perfectly valid point, and please be picky